packages/api/tests/es/test-stats.js

```diff
@@ -0,0 +1,132 @@
+'use strict';
+
+const test = require('ava');
+const rewire = require('rewire');
+const get = require('lodash.get');
+
+const awsServices = require('@cumulus/aws-client/services');
+const s3 = require('@cumulus/aws-client/S3');
+const { randomString } = require('@cumulus/common/test-utils');
```

Comment:
I don't know where my last comment went, but will you use randomId() instead?  (just be sure to use all lowercase on your bucket name e.g. `randomId('systembucket')`.  Thanks.  It makes it easier to debug.